<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>601</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_label2">
   <property name="geometry">
    <rect>
     <x>729</x>
     <y>40</y>
     <width>611</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="test_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="take_pic">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>981</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>341</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Focus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="h_min">
          <property name="maximum">
           <number>179</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Exposure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="s_min">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Zoom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="v_min">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <zorder>take_pic</zorder>
  <zorder>start_button</zorder>
  <zorder>stop_button</zorder>
  <zorder>test_button</zorder>
  <zorder>load_button</zorder>
  <zorder>image_label2</zorder>
  <zorder>image_label1</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
